docs/self-managed/components/orchestration-cluster/zeebe/exporters/elasticsearch-exporter.md

```diff
@@ -12,6 +12,11 @@ import TabItem from "@theme/TabItem";
 Please refer to [supported environments](/reference/supported-environments.md#camunda-8-self-managed) to find out which versions of Elasticsearch are supported in a Camunda 8 Self-Managed setup.
 :::
 
+:::note
+As of **8.8**, Camunda uses the **Camunda Exporter** after all **≤8.7** records have been exported and consumed.  
+The Elasticsearch and OpenSearch exporters remain available for existing setups and custom use cases until the migration is complete.
```

Comment:
I find this wording a bit off, because the exporters are usable after the migration is complete too, just that the Orchestration cluster will stop consuming records after 8.7. For Optimize and custom use cases, this goes long beyond the migration.

Worst case scenario someone could read this and either think that they exporters can be removed (and this might not be true), or that they incorrectly assume that those records stopped getting consumed (which for Optimize is not true)

Reply:
Adjusted in incoming commit 👍